ginh.sh

```diff
@@ -175,7 +179,9 @@ fi
 
 filters+=("fish_filter")
 filters+=("zsh_extended_filter")
-filters+=("reverse_aliases_filter")
+if [ $alias -eq 1 ]; then
```

Comment:
would it be cleaner to instead only set alias to a value when the ginh's -a flag is used, and then here check if alias is set with bash's -z flag?

Reply:
yep, adding